src/__tests__/http-test.ts

```diff
@@ -491,8 +490,8 @@ function runTests(server: Server) {
       const app = server();
 
       // Middleware that adds req.foo to every request
-      app.use((req, _res, next) => {
-        req.foo = 'bar';
+      app.use((req: Request, _res: Response, next: Next) => {
```

Comment:
Do you need to explicitly state types here?
I thought TS will infer it from `app.use`.